src/simplewallet/simplewallet.cpp

```diff
@@ -4939,7 +4939,7 @@ bool simple_wallet::sweep_unmixable(const std::vector<std::string> &args_)
 //----------------------------------------------------------------------------------------------------
 bool simple_wallet::sweep_main(uint64_t below, const std::vector<std::string> &args_)
 {
-  // sweep_all [index=<N1>[,<N2>,...]] [<ring_size>] <address> [<payment_id>]
+  // sweep_all [index=<N1>[,<N2>,...]] [<priority>] [<ring_size>] [outputs=<N>] <address> [<payment_id>]
```

Comment:
Realized just now: please reflect this change in the `help` command as well.

Reply:
Ok, I'll do it in a few hours. I would also like to rebase on master after GCC8 fixes to make sure I didn't miss anything during previous rebase.